viewer/src/utils/CogniteClient3dV2Extensions.ts

```diff
@@ -0,0 +1,115 @@
+/*!
```

Comment:
Perhaps scope the file to Model3D in the name as well, in case we need more over time?

`CogniteClientModel3DV2Extensions`

Perhaps also drop the V2 in the name? I suppose extensions can just be non-versioned and work with whichever API version is necessary.

Reply:
I agree with dropping V2, but I think 3d is sufficient - why is adding Model necessary?